documentation/modules/cruise-control/con-optimization-proposals.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// assembly-cruise-control-concepts.adoc
+
+[id='con-optimization-proposals-{context}']
+
+= Optimization proposals overview
+
+An _optimization proposal_ is a summary of proposed changes that would produce a more balanced Kafka cluster, with partition workloads distributed more evenly among the brokers. 
+Each optimization proposal is based on the set of xref:con-optimization-goals-{context}[optimization goals] that were used to generate it.
+
+Optimization proposals always satisfy all the configured hard goals, although not necessarily all the soft goals. Hard goals are ignored if `skipHardGoalCheck: true` is specified in the `KafkaRebalance` custom resource.
```

Comment:
This statement is not correct. The `skipHardGoalCheck: true` only applies to the check that is done for the user defined goals and weather they contain all the specified `hard.goals`. If a user sets `skipHardGoalCheck: true` all the user defined goals that are in the `hard.goals` list will be treated as hard goals.

Reply:
You can just remove the second sentence.

Reply:
Right. I'm going to document this in the "Hard goals" section instead.

Reply:
I added a note.